source/index.js

```diff
@@ -56,6 +56,10 @@ document.body.appendChild(renderer.view)
 // load these textures from our `images` directory.
 var RED_TEXTURE = Pixi.Texture.fromImage(require("images/red-starship.png"))
 var BLUE_TEXTURE = Pixi.Texture.fromImage(require("images/blue-starship.png"))
+var GREEN_TEXTURE = Pixi.Texture.fromImage(require("images/green-starship.png"))
+var YELLOW_TEXTURE = Pixi.Texture.fromImage(require("images/yellow-starship.png"))
+
+var spaceshipTextures = [RED_TEXTURE, BLUE_TEXTURE, GREEN_TEXTURE, YELLOW_TEXTURE]
```

Comment:
Maybe we could use this like a queue, so whatever texture is at the zero index is what we assign to the `this.texture`.

```js
    // Remove the first element and put it at the end.
    spaceshipTextures.push(spaceshipTextures.shift())
    this.texture = spaceshipTextures[0]
```